backend/src/main/java/ca/bc/gov/restapi/results/oracle/dto/OpeningSearchFiltersDto.java

```diff
@@ -82,6 +84,28 @@ public OpeningSearchFiltersDto(
         Objects.isNull(mainSearchTerm) ? null : mainSearchTerm.toUpperCase().trim();
   }
 
+  // Create a constructor with only the List<String> openingIds
+  public OpeningSearchFiltersDto(
```

Comment:
You can make use of lombok `@AllArgsConstructor` annotation on this class instead.

Reply:
https://github.com/bcgov/nr-silva/pull/431/commits/6d5c5cf160fe3729b42ccc1f79b656616f234e37
fixed in above commit